lib/pages/signup/start-page.js

```diff
@@ -14,52 +14,22 @@ export default class StartPage extends AsyncBaseContainer {
 		return await this.setABTestControlGroupsInLocalStorage();
 	}
 
-	static getStartURL( {
-		culture = 'en',
-		flow = '',
-		domainFirst = false,
-		domainFirstDomain = '',
-		query = ''
-	} = {} ) {
-		let url;
-		let queryStrings = '';
-		url = dataHelper.configGet( 'calypsoBaseURL' ) + '/start';
+	static getStartURL( { culture = 'en', flow = '', query = '' } = {} ) {
+		let route = 'start';
+		let queryStrings = [];
 
 		if ( flow !== '' ) {
-			url += '/' + flow;
-		}
-
-		if ( domainFirst === true ) {
```

Comment:
I'm passing domain first as the flow now and using `query` which is now supported 🎉